ovs/v1/ovs.yaml

```diff
@@ -242,39 +242,15 @@ paths:
           application/json:
             schema:
               allOf:
-                - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/EVENT_DOMAIN/1.0.0#/components/schemas/transportEventBody'
-                - type: object
-                  properties:
-                    facilityType:
-                      description: |
-                        The code to identify the specific type of facility.
-                        - PBPL (Pilot boarding place)
-                        - BRTH (Berth)
-                      maxLength: 4
-                      example: PBPL
-                      enum:
-                        - PBPL
-                        - BRTH
+                - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/EVENT_DOMAIN/1.0.0#/components/schemas/operationsEventBody'
       responses:
         '201':
           description: Transport-call successfully posted
           content:
             application/json:
               schema:
                 allOf:
-                  - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/EVENT_DOMAIN/1.0.0#/components/schemas/transportEvent'
```

Comment:
Removed overwriting of facility Type, as this is set correct on OperationalEvent